fedn/fedn/network/combiner/aggregators/aggregatorbase.py

```diff
@@ -116,11 +116,17 @@ def load_model_update(self, model_update, helper):
         model_id = model_update.model_update_id
         model = self.round_handler.load_model_update(helper, model_id)
         # Get relevant metadata
-        data = json.loads(model_update.meta)['training_metadata']
-        config = json.loads(json.loads(model_update.meta)['config'])
-        data['round_id'] = config['round_id']
-
-        return model, data
+        metadata = json.loads(model_update.meta)
+        if 'config' in metadata.keys():
+            # Used in Python client
+            config = json.loads(metadata['config'])
```

Comment:
Why do we need to support both versions of the message? Why not just introduce the new config?

Reply:
because of backward compability